src/components/policy/src/policy/policy_table/table_struct/types.cc

```diff
@@ -489,6 +489,24 @@ ModuleConfig::ModuleConfig(const Json::Value* value__)
     vehicle_model(impl::ValueMember(value__, "vehicle_model")),
     vehicle_year(impl::ValueMember(value__, "vehicle_year")) {
 }
+
+void ModuleConfig::SafeCopyFrom(const ModuleConfig& from) {
```

Comment:
``operator=`` must copy _all_ fields of this strucs.
In case it meet requirements - it's better way to use operator.
If not - agree we this implementation.